### PR TITLE
Added Ebola risk map and updated 'risk' specific content in Sidebar

### PR DESCRIPTION
**Pull request by fernandodonaire**
Screen Shot 2020-11-13 at 10 52 54 AM

Can be tested by: 

1) Pulling down this branch.
2) Running the app locally. 
3) On the homepage, clicking on the `risk` button at the top of the map.

This will display the fullscreen `risk map` will change the content that is displayed in the Sidebar.

### CODE REVIEW

File: src/components/Sidebar/EbolaRiskList.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { BlockPadded } from "../styled-components/Block";
+
+const EbolaRiskList = () => (
+  <BlockPadded>
+    <strong>
+      <p>
+        The top 10 countries with the highest relative risk of Ebola spread are:
```
Comment (SpencerMarks-IDHA) [inline src/components/Sidebar/EbolaRiskList.js:8]: do we really want to mix JS and HTML like this?

**Review by SpencerMarks-IDHA — COMMENTED** [review]
minor concern with hardcoding html in js code,
&
nondynamic list of countries but will defer to you

**Review by SpencerMarks-IDHA — APPROVED** [review]

File: src/components/Sidebar/EbolaRiskList.js (hunk 1 above)
Comment (fernandodonaire) [reply src/components/Sidebar/EbolaRiskList.js:8]: Hey @SpencerMarks-IDHA , that looks like HTML but it's actually JSX which is a Javascript syntax to create React elements: https://reactjs.org/docs/introducing-jsx.html

**Comment by fernandodonaire** [comment]
Thanks for the review @SpencerMarks-IDHA !

**Pull request merged**